src/Language/PureScript/Errors.hs

```diff
@@ -636,6 +638,11 @@ prettyPrintSingleError (PPEOptions codeColor full level showWiki) e = flip evalS
         where
         go TUnknown{} = True
         go _ = False
+    renderSimpleErrorMessage (AmbiguousTypeVariables t _) =
+      paras [ line "The inferred type"
+            , indent $ line $ markCode $ prettyPrintType t
```

Comment:
I just realized this should use `typeAsBox`, since this way, the indentation gets messed up if the type takes up multiple lines.

I can put together a PR for this later, unless you want to take a look @bbqbaron ?